src/atlas_assistant/api.py

```diff
@@ -106,3 +120,44 @@ async def login(
             detail="Incorrect username or password",
             headers={"WWW-Authenticate": "Bearer"},
         )
+
+
+@app.post("/chat", tags=["chat"])
+async def chat(
+    chat_request: ChatRequest,
+    settings: Annotated[Settings, Depends(get_settings)],
+    current_user: Annotated[User, Depends(get_current_user)],  # pyright: ignore[reportUnusedParameter]
+) -> StreamingResponse:
+    agent = create_agent(settings)
+    thread_id = uuid.uuid4()
+    return StreamingResponse(
+        query_agent(agent, chat_request.query, str(thread_id), settings)
+    )
```

Comment:
```suggestion
@app.post("/chat", tags=["chat"])
async def chat(
    chat_request: ChatRequest,
    settings: Annotated[Settings, Depends(get_settings)],
    current_user: Annotated[User, Depends(get_current_user)],  # pyright: ignore[reportUnusedParameter]
) -> StreamingResponse:
    agent = create_agent(settings)
    thread_id = uuid.uuid4()
    return StreamingResponse(
        query_agent(agent, chat_request.query, str(thread_id), settings),
        media_type="text/event-stream",
    )
```

Reply:
A quick search indicates to me it should be `application/x-ndjson`?

https://stackoverflow.com/questions/52098863/whats-the-difference-between-text-event-stream-and-application-streamjson

Reply:
we can start with just ndjson. i think what the additional feature we'd get is like true streaming / automatic reconnection in the UI which may become a UX requirement at some point?

--- 
another thought though is if the goal is just to support swagger why not both? 

```
accept = request.headers.get("accept", "")
    if "text/event-stream" in accept:
      # event 
    else:
      # ndjosn 
```